client/src/App.js

```diff
@@ -1,3 +1,18 @@
+import Card from './components/Card'
+import styled from 'styled-components'
+
 export default function App() {
-  return <div>Hello world</div>
+  return (
+    <Container>
+      <Card>What does Mongo mean?</Card>
+      <Card>What does Mongo mean?</Card>
+      <Card>What does Mongo mean?</Card>
```

Comment:
Why not create an array with mock data and map over it? That way you'd already implement it as you would later on when you get more data.